modules/manipulateGTC.py

```diff
@@ -174,6 +208,44 @@ def snpOverride(manifest, overrides):
 ###################################################################################################################
     logger.debug('Preparing to read in bpm file...')
     manifest = BeadPoolManifest(bpm)
+    def get_manifest_csv():
```

Comment:
Just for internal doc and as a reminder to myself; this code chunk is to support the input csv as the manifest instead of the bpm